packages/pass-style/src/deeplyFulfilled.js

```diff
@@ -6,8 +6,9 @@ import { passStyleOf } from './passStyleOf.js';
 import { makeTagged } from './makeTagged.js';
 
 /**
- * @import {Passable, Primitive, CopyRecord, CopyArray, CopyTagged, RemotableObject} from '@endo/pass-style'
+ * @import {Passable, CopyRecord, CopyArray, CopyTagged, RemotableObject} from '@endo/pass-style'
  */
+/** @import {Callable, RemotableBrand} from '@endo/eventual-send' */
```

Comment:
Nice. I'lll just mention that I'm a bit surprised that `RemotableBrand` is defined by @endo/eventual-send but `RemotableObject` is defined by @endo/pass-style. No change suggested.

Reply:
Yeah they're very different, somewhat orthogonal things. I've been discussing with @michaelfig, and from eventual-send's pov, the target or arguments don't require any specific pass-style. `RemotableBrand` helps the typing to capture the shape of remote methods (and the shape of local data) without exposing any actual property on the type itself (its storing them in the private fields of a "stamped" class).

`RemotableObject` is the pass-style brand, which includes the 2 symbol properties (`PASS_STYLE` and `Symbol.toStringTag`), denoting a remotable in passables.